src/System.Net.Http.WinHttpHandler/src/System.Net.Http.WinHttpHandler.csproj

```diff
@@ -7,6 +7,8 @@
     <OutputType>Library</OutputType>
     <AssemblyName>System.Net.Http.WinHttpHandler</AssemblyName>
     <GeneratePlatformNotSupportedAssembly Condition="'$(TargetsUnix)' == 'true'">true</GeneratePlatformNotSupportedAssembly>
+    <!-- Although we have a NS configuration, the actual UWP implementation is a shim over WinRT: so just validate against OneCore -->
```

Comment:
>So then the plan is to not support this library on UWP? If so how do we plan to prevent folks form using it there and then failing the store ingestion? What platforms do we plan to support this library on?

This library is only meant for Windows platform.  Not *Nix.  It is only meant for .NET Core (non-UWP) or .NET Framework (Desktop).